appveyor.yml

```diff
@@ -26,7 +26,7 @@
         - CHANGELOG.md
         # These files are only used by the Linux CI builds and don't affect
         # the Windows builds.
-        - .github/**
+        - .github/**/*
```

Comment:
Confirming that the extra `/*` at the end is what the [filtering rules](https://www.appveyor.com/docs/how-to/filtering-commits/#file-matching-rules) say should be used to match all files in all directories recursively.